platform/core/src/services/ViewportGridService/ViewportGridService.ts

```diff
@@ -45,8 +45,8 @@ class ViewportGridService extends PubSubService {
     return this.presentationIdProviders.get(id);
   }
 
-  public setIsReferenceViewable(viewportId: string, isReferenceViewable: boolean): void {
-    this.serviceImplementation._setIsReferenceViewable(viewportId, isReferenceViewable);
+  public isReferenceViewable(viewportId, reference, options) {
```

Comment:
seems like a migration guide is needed, it was before public setIsReferenceViewable